src/azure-cli/azure/cli/command_modules/cdn/tests/latest/afdx_scenario_mixin.py

```diff
@@ -18,7 +18,7 @@ def _add_paramter_if_needed(command, paramter_name, parameter_value):
 class CdnAfdScenarioMixin:
     def afd_profile_create_cmd(self, resource_group_name, profile_name, tags=None, checks=None, options=None,
                                sku="Standard_AzureFrontDoor", expect_failure=False):
-        command = f'afd profile create -g {resource_group_name} --profile-name {profile_name} --sku {sku}'
+        command = f'afd profile create -g {resource_group_name} --profile-name {profile_name} --sku {sku} --location global'
```

Comment:
It's not a good practice to expose extra parameters, there are many code snippets working as "force location to global", you can refer to them.